src/test/java/info/ludwikowski/fluentbuilder/sample/types/GenericType.java

```diff
@@ -0,0 +1,5 @@
+package info.ludwikowski.fluentbuilder.sample.types;
```

Comment:
I would create `generics` package instead of `types` and put there: `GenericType`, `SimpleType`, `ClassWithGenericTypes`, `ClassWithoutDefaultConstructorWithGenericTypes`